examples/go/bulk-operations/main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/oapi-codegen/runtime/types"
+
+	"github.com/hatchet-dev/hatchet/pkg/client/rest"
+	v1 "github.com/hatchet-dev/hatchet/pkg/v1"
+)
+
+func main() {
+	// > Setup
+	hatchet, err := v1.NewHatchetClient()
+	if err != nil {
+		log.Fatalf("failed to create hatchet client: %v", err)
+	}
+
+	ctx := context.Background()
+
+	workflows, err := hatchet.Workflows().List(ctx, nil)
+	if err != nil {
+		log.Fatalf("failed to list workflows: %v", err)
+	}
+
+	if workflows == nil || workflows.Rows == nil || len(*workflows.Rows) == 0 {
+		log.Fatalf("no workflows found")
+	}
+
+	selectedWorkflow := (*workflows.Rows)[0]
+	selectedWorkflowUUID := uuid.MustParse(selectedWorkflow.Metadata.Id)
+
```

Comment:
I think these sections will need closing indicators (`> !!`) to know the snippet has ended / so they generate properly for us to use in the docs

Reply:
although looking at what's generated, maybe not? idk 🤷

Reply:
ooooh nvm this _is_ the generated stuff 🤦 ignore me, sorry!